swift_browser_ui_frontend/package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.4.0",
```

Comment:
reminder: version bump would be nice when merging in `master`

Reply:
Yes, that's going to happen when the changes are merged from devel->master :+1: